packages/react-compose/src/next/types.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * Generic name to any dictionary.
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export type GenericDictionary = Record<string, any>;
+
+export interface ComponentProps extends GenericDictionary {
+  as?: React.ElementType;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  ref?: React.Ref<any>;
+  className?: string;
+}
```

Comment:
Are these duplicates from `src/types.ts` purposeful?

Reply:
My plan was to replace the files in the src folder completely, so i started with the types.ts file content and started purging as needed.

Reply:
The whole folder is temporary, so i'll leave for now.